README.md

```diff
@@ -10,11 +10,368 @@
+type, the labels used to slice the metric, and a description.
+
+| Metric Name      | Metric Type   | Labels              | Description                                  |
+| ---------------- |---------------|---------------------|:---------------------------------------------|
+| `sth_timestamp` | GaugeVec | `uri` | Timestamp of fetched STH |
+| `sth_age`       | GaugeVec | `uri`            | Elapsed time since timestamp of fetched STH |
+| `sth_failures`  | CounterVec | `uri` | Count of failures fetching a STH |
+| `sth_latency`   | HistogramVec | `uri` | Latency of fetching a STH |
+| `sth_proof_latency` | HistogramVec | `uri` | Latency of fetching a STH consistency proof |
+| `sth_inconsistencies` | CounterVec | `uri`, `type` | Count of instances two STHs could not be proved consistent |
+| `cert_submit_latency` | HistogramVec | `uri`, `precert` | Latency from submitting a cert or precert |
+| `cert_submit_results` | CounterVec | `uri`, `status`, `precert`, `duplicate` | Result from submitting a cert or precert |
+| `cert_storage_failures` | CounterVec | `uri`, `type` | Count of instances a cert/SCT couldn't be saved to the local DB to watch for inclusion |
+| `stored_scts` | CounterVec | `uri` | Count of unique cert/SCTs retrieved and stored in the db |
+| `oldest_unincorporated_cert` | GaugeVec | `uri` |Number of seconds since the oldest cert waiting on incorporation was submitted |
+| `unincorporated_certs` | GaugeVec | `uri` | Number of certs/SCTs submitted but not yet incorporated |
+| `inclusion_checker_errors` | CounterVec | `uri`, `type` | Number of errors encountered attemtping to check for cert inclusion |
+
+* Possible `sth_inconsistency` `type` values are:
+  * `"equal-treesize-inequal-hash"` for when two STH's have the same treesize and different hashes.
+  * `"failed-to-get-proof"` for when an error occurs fetching the consistency proof.
+  * `"failed-to-verify-proof"` for when a returned STH consistency proof can't
+  be validated.
+
+* Possible `cert_submit_results` `status` values are:
+  * `"fail"` for failed submissions.
+  * `"ok"` for successful submissions.
+
+* `cert_submit_results` will have a true `precert` label when the submission was a precert.
+
+* `cert_submit_results` will have a true `duplicate` label when the submission was a resubmission of a previously submitted cert/precert.
+
+* Possible `cert_storage_failures` `type` values are:
+  * `"marshalling"` for failures to marshal a returned SCT for storage.
+  * `"storing"` for failures to insert the cert/SCT into the DB.
+
+* Possible `inclusion_checker_errors` `type` values are:
+  * `"getIndex"` for failures to get the current stored tree index from the DB.
+  * `"getUnseen"` for failures to find unseen certs/SCTs in the DB.
+  * `"getSTH"` for failures to fetch an STH to determine entries needing to be
+  fetched.
+  * `"getEntires"` for failures to get entries from the log.
+  * `"checkEntires"` for failures to check unseen certs against the returned new
```

Comment:
```suggestion
  * `"checkEntries"` for failures to check unseen certs against the returned new
```

Reply:
How did I make the same typo twice :laughing: